packages/edge-login-ui-rn/src/components/screens/RecoveryLoginScreen.tsx

```diff
@@ -10,7 +10,7 @@ import { Dispatch, RootState } from '../../types/ReduxTypes'
 import { LoginAttempt } from '../../util/loginAttempt'
 import { Button } from '../common/Button'
 import { FormField } from '../common/FormField'
-import { Header } from '../common/Header'
+import { HeaderComponent } from '../common/Header'
```

Comment:
Can we rename the component to `Header`, so the change stays local to the file instead of leaking out to all these other files?